src/main.c

```diff
@@ -22,7 +22,11 @@ int main(int argc, char **argv)
         else die("Incorrect usage! Use\n\t--no-shell-escape\n\t--no-prompt");
     }
 
-    for(;;){
+    char *p;
+    int c,b,l,d;
```

Comment:
While you’re at it, also fix the whitespacing according to the convention in the code.

`int c, b, l, d;`

`for(;;){`